linux-4k/PKGBUILD

```diff
@@ -0,0 +1,206 @@
+# POWER Maintainer: Alexander Baldeck <alex.bldck@gmail.com>
+# Maintainer: Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>
+
+pkgbase=linux
```

Comment:
This should also be linux-4k to not conflict with the default linux package.

Reply:
I thought I already did that o_o
Done